libr/core/cmd_type.c

```diff
@@ -61,7 +61,7 @@ static int sdbdeletelink(void *p, const char *k, const char *v) {
 }
 static int linklist(void *p, const char *k, const char *v){
 	if(!strncmp(k,"link.",strlen("link.")))
```

Comment:
too many strlens of the same thing use `const int link_len = strlen ("link.");` or so